File: 004_reading_and_writing_a_mesh/solution/write_tetrahedron.m
function write_tetrahedron()
%WRITE_TETRAHEDRON  write a tetrahedron to the file "tetrahedron.obj".
%
% write_tetrahedron();
%

V = [0, 0, 0; 1, 0, 0; 0, 1, 0; 0, 0, 1];
F = [1, 2, 3; 1, 2, 4; 2, 3, 4; 3, 1, 4];
writeOBJ('tetrahedron.obj', V, F);

end

File: 004_reading_and_writing_a_mesh/solution/write_tetrahedron.py
import numpy as np, gpytoolbox as gpy

def write_tetrahedron():
    """
    Writes a tetrahedron to the file "tetrahedron.obj"
    """

    V = np.array([[0., 0., 0.], [1., 0., 0.], [0., 1., 0.], [0., 0., 1.]])
    F = np.array([[1,0,2], [0,1,3], [1,2,3], [2,0,3]])
    gpy.write_mesh("tetrahedron.obj", V, F)

